tasks/install_jq.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install jq
+  apt:
+    name: jq
+    state: latest
+    update_cache: true
+    cache_valid_time: 600
```

Comment:
We are using [jq](https://stedolan.github.io/jq/) to query the JSON and search for parameter contexts by name.  Ansible natively supports [jmespath](https://jmespath.org/), but I was not able to find a query that returns what we want.